CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contribute @ howtolearn
+
+At howtolearn, we are open for everyone who wants to share his knowledge with the community.
+For a successful contribution, there are only some little steps to follow.
+
+## 1) Write your personal TOP 5 of learning resources for one topic
+For our learning paths we need only 5 resources, so there is not too much input at once for a newbie in a topic.
```

Comment:
Hier würd ich noch einen Satz zur Reihenfolge unterbringen, also dass (in der Regel) mit 1 angefangen werden soll und nicht einfach mit irgendeiner der 5 Quellen. Vielleicht so:
```suggestion
For our learning paths we need only 5 resources, so there is not too much input at once for a newbie in a topic. Since it's a path, you maybe want to suggest the source how to start (your Nr. 1), how to proceed (Nr. 2) and so on. Then just tell in your comment that your order is intentional. Otherwise comment somewhere that your order is random. 
```